### DISCUSSION TITLE
r/city-china-zhejiang-huzhou-changxing-baduka

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#69] [Announcements]
# r/city-china-zhejiang-huzhou-changxing-baduka

去湖州长兴玩儿水，不要只去川步村，这里人少景还美，配套也齐全，停车还免费，是个更好的小众旅行玩儿水的宝藏目的地

https://blog.ops-coffee.cn/r/city-china-zhejiang-huzhou-changxing-baduka

**Comment by xyzbz** [comment]
泡面下次要加香肠。

**Reply by ops-coffee** [reply]
我不喜欢加香肠，但必须得加鸡蛋，鸡蛋是灵魂
